lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -65,43 +72,131 @@ class UnusedJavaScript extends ByteEfficiencyAudit {
     }
 
     return {
+      unusedByIndex,
       unusedLength: unused,
       contentLength: maximumEndOffset,
     };
   }
 
   /**
-   * @param {Array<{unusedLength: number, contentLength: number}>} wasteData
-   * @param {LH.Artifacts.NetworkRequest} networkRecord
+   * @param {LH.Audit.ByteEfficiencyItem} item
+   * @param {WasteData[]} wasteData
+   * @param {LH.Artifacts.Bundle} bundle
+   * @param {ReturnType<typeof UnusedJavaScript.determineLengths>} lengths
+   * @param {number} bundleSourceUnusedThreshold
+   */
+  static createBundleMultiData(item, wasteData, bundle, lengths, bundleSourceUnusedThreshold) {
```

Comment:
iirc you had more descriptive function names before.  

i think this should be two functions.. 
one is L92 - 118... and titled around `computing bundle waste` ish
and then a second is L120 - 138 and calling that `bundle subrow data` ish  sgtm

you said you're about to extract some of this into a computed artifact, so it sounds like you're going to do this in your next PR anyhow, yeah?

Reply:
yeah, I have a few better names and split this fn in a computed artifact PR :)